README.md

```diff
@@ -18,17 +18,27 @@ A repository for hosting [Nextflow DSL2](https://www.nextflow.io/docs/latest/dsl
 
 ## Table of contents
 
-- [Using existing modules](#using-existing-modules)
-- [Adding a new module file](#adding-a-new-module-file)
-    - [Checklist](#checklist)
-    - [nf-core modules create](#nf-core-modules-create)
-    - [Test data](#test-data)
-    - [Running tests manually](#running-tests-manually)
-    - [Uploading to `nf-core/modules`](#uploading-to-nf-coremodules)
-    - [Guidelines](#guidelines)
-- [Terminology](#terminology)
-- [Help](#help)
-- [Citation](#citation)
+
+- [!nf-core/modules](#img-srcdocsimagesnfcore-modules_logopng-altnf-coremodules)
+    - [Table of contents](#table-of-contents)
```

Comment:
Why do we need these? The top one looks broken?

Reply:
I have absolutely no where that has come from!? Maybe an autoformatting thing in VSCode :facepalm: